### DISCUSSION TITLE
How to setup Cloudflare with Proxy?

### DISCUSSION THREAD

**Discussion by alexbjorlig** [vercel/community#553] [Help] [answered]
Hi there. I'm pretty new to Vercel. I hope someone out there can help me with the following:

1. Setup Cloudflare DNS **with** Proxy (so I can enjoy Cloudflare analytics)
2. Optionally use Cloudflare caching - so there is no "extra" hop on the edge when users request HTML
3. I know that I would then have to bust the Cloudflare cache manually, but I think that's ok

When I enable cloudflare proxy, I get this warning:
![Screen Shot 2022-04-10 at 11 23 30](https://user-images.githubusercontent.com/9967422/162611596-d9c34ce1-b7fc-4f47-82a6-4c9a456a9bd9.png)

In Cloudflare I have enabled strict SSL
![Screen Shot 2022-04-10 at 11 24 48](https://user-images.githubusercontent.com/9967422/162611625-d8a5d4ed-fcab-434a-88dc-f5710654d3ab.png)

And my `vercel.json` file is empty. The underlying project is written in sveltekit.

When I request `https://21risk.com` I get a status code 308, `cf-cache-status: DYNAMIC`. Do I need to do something on Cloudflares configuration to cache the content?

**Comment by swarnava** [comment, marked as answer]
Hi there,

We have a helpcenter article that will help you to get started: https://vercel.com/support/articles/using-cloudflare-with-vercel
